python/ray/experimental/state.py

```diff
@@ -1310,8 +1311,10 @@ def cluster_resources(self):
         else:
             clients = self.client_table()
             for client in clients:
-                for key, value in client["Resources"].items():
-                    resources[key] += value
+                # Only count resources from live clients.
+                if client["IsInsertion"]:
+                    for key, value in client["Resources"].items():
+                        resources[key] += value
```

Comment:
can a `IsInsertion=False` entry have resources?

Reply:
Well it means that the node has been removed so it's resources are not relevant anymore.